src/UI/Header.module.css

```diff
@@ -6,7 +6,6 @@
 	background-color: var(--parley-nav-background-color, #4A5E83);
 	border-radius: 5px 5px 0 0;
 	box-shadow: 0 1px 3px 0 rgba(74, 94, 131, 0.07);
-	color: var(--parley-nav-text-color, #FFF);
```

Comment:
Dit is niet global styles maar wel bewust verwijderd?

Reply:
Ja deze var bestaat niet